scripts/wipe_sensitive_info_db.py

```diff
@@ -10,64 +10,64 @@
 import sys
 
 flags.DEFINE_boolean("quiet", False, "Do not prompt user confirmation")
-
+flags.DEFINE_boolean(
+  "keep_beds", False, "Whether to keep bed occupation data."
+)
 FLAGS = flags.FLAGS
 
 ALTERATIONS = {
-    "bed_counts": [
-        ("n_covid_free", 2),
-        ("n_covid_occ", 4),
-        ("n_ncovid_free", 3),
-        ("n_ncovid_occ", 5),
-        ("n_covid_deaths", 0),
-        ("n_covid_healed", 0),
-        ("n_covid_refused", 0),
-        ("n_covid_transfered", 0),
-        ("message", '"Have a nice day!"'),
-    ],
-    "icus": [
-        ("telephone", "33120000000 + icus.icu_id")
-    ],  # we reuse the ID to respect telephone uniqueness constraint
-    "users": [
-        ("name", '"Jean Dumont"'),
-        ("email", '"jean.dumont@example.org"'),
-        ("telephone", "33600000001 + users.user_id"),
-        ("password_hash", 42),
-    ],
-    "external_clients": [
-        ("name", '"Jean Dumont"'),
-        ("email", '"jean.dumont@example.org"'),
-        ("telephone", "33600000001 + external_clients.external_client_id"),
-    ],
+  "bed_counts": [
+    ("n_ncovid_free", 3),
+    ("n_ncovid_occ", 5),
+    ("n_covid_deaths", 0),
+    ("n_covid_healed", 0),
+    ("n_covid_refused", 0),
+    ("n_covid_transfered", 0),
+    ("message", '"Have a nice day!"'),
+  ],
+  "icus": [("telephone", "33120000000 + icus.icu_id")
+           ],  # we reuse the ID to respect telephone uniqueness constraint
+  "users": [
+    ("name", '"Jean Dumont"'),
+    ("email", '"jean.dumont@example.org"'),
+    ("telephone", "33600000001 + users.user_id"),
+    ("password_hash", 42),
+  ],
+  "external_clients": [
+    ("name", '"Jean Dumont"'),
+    ("email", '"jean.dumont@example.org"'),
+    ("telephone", "33600000001 + external_clients.external_client_id"),
+  ],
 }
 
 
 def wipe_db(path):
-    conn = sqlite3.connect(path)
-    cur = conn.cursor()
-
-    for alteration in ALTERATIONS.items():
-        affectations = ",".join([f"{k} = {v}" for (k, v) in alteration[1]])
-        query = f"UPDATE {alteration[0]} SET {affectations};"
-        print(query)
-        conn.execute(query)
-    conn.commit()
-    conn.close()
+  conn = sqlite3.connect(path)
+  cur = conn.cursor()
+  if FLAGS.keep_beds:
+    ALTERATIONS["bed_counts"].extend([("n_covid_free", 2), ("n_covid_occ", 4)])
+  for alteration in ALTERATIONS.items():
+    affectations = ",".join([f"{k} = {v}" for (k, v) in alteration[1]])
+    query = f"UPDATE {alteration[0]} SET {affectations};"
+    print(query)
```

Comment:
The print statement should also probably be in a `if not FLAGS.quiet:` guard but maybe that's a separate PR?

Reply:
Done.